README.md

```diff
@@ -214,18 +214,19 @@ We're going to try spinning this out into its own library; see: https://github.c
       attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, <a href="http://www.openinframap.org/about.html">About OpenInfraMap</a>'
     });   
 
-## Add Leaflet-FullHash for easy sharing of map :    
+## Add hash support for easy sharing of map :    
 
 ### Steps To add :
 
 1.) Add link :
 
-    <script src="../node_modules/leaflet-fullhash/leaflet-fullHash.js"></script>
+       <script src="../lib/leaflet-fullUrlHash.js"></script>
```

Comment:
I am not sure if I am being picky about the extra space added before the script tag. Other than that the changes look good to me. What do you think @sagarpreet-chadha?